src/main/java/seedu/address/logic/commands/contact/CAddCommand.java

```diff
@@ -18,22 +20,26 @@
  */
 public class CAddCommand extends Command {
 
-    public static final String COMMAND_WORD = "add";
+    public static final String COMMAND_WORD = "cadd";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a contact to the address book. "
             + "Parameters: "
             + PREFIX_NAME + "NAME "
-            + PREFIX_PHONE + "PHONE "
             + PREFIX_EMAIL + "EMAIL "
-            + PREFIX_ADDRESS + "ADDRESS "
+            + "[" + PREFIX_PHONE + "PHONE_NUMBER] "
```

Comment:
Should we order the parameters to be the same as the below ones? to add consistency?